packages/plugin-snowplow/src/androidMain/kotlin/ly.iterative.itly.snowplow/SnowplowPlugin.kt

```diff
@@ -0,0 +1,57 @@
+/**
+ * Android SnowplowPlugin
+ */
+package ly.iterative.itly.snowplow
+
+import ly.iterative.itly.*
+
+import com.snowplowanalytics.snowplow.Snowplow;
+import com.snowplowanalytics.snowplow.controller.TrackerController
+import com.snowplowanalytics.snowplow.event.SelfDescribing
+import com.snowplowanalytics.snowplow.payload.SelfDescribingJson
+
+actual class SnowplowPlugin actual constructor(
+    options: SnowplowOptions
+) : Plugin(ID) {
+    companion object {
+        @JvmField
+        val ID = "snowplow"
+    }
+
+    private var config: SnowplowOptions = options
+    private lateinit var logger: Logger
+    private var snowplow: TrackerController? = null
+
+    val client: TrackerController?
+        get() = this.snowplow
+
+    override fun load(options: PluginLoadOptions) {
+        logger = options.logger
+        logger.debug("[plugin-snowplow] load")
+
+        if (config.tracker != null) {
```

Comment:
nit picking but I like single assignment when possible
```
this.snowplow = config.tracker ?:  Snowplow.getDefaultTracker()
```

Reply:
Good correction. I'm new to Kotlin actually, not familiar with its style